src/engine/qcommon/q_math.cpp

```diff
@@ -187,7 +187,7 @@ signed char ClampChar( int i )
 }
 
 // this isn't a real cheap function to call!
-int DirToByte( vec3_t dir )
+int DirToByte( vec3_t const dir )
```

Comment:
Open question: doesn't this break compat between older game and newer daemon, and newer game and old daemon?

Reply:
Making constness explicit does not break compat.

Reply:
I've never studied ABI compat, but I would guess that since it's possible to define both `f(int*)` and `f(const int*)` at the same time, the function names must be mangled differently

Reply:
(I'll defer to @slipher on that one)

Reply:
There is no ABI boundary here, or anywhere in normal C++ functions. Our binary interfaces are the IPC serialized stuff and anything going on the network.